### PR TITLE
migration to create NewsPost table

### PR DESCRIPTION
**Pull request by amanpatel123**
### What are you trying to accomplish?
- First step towards converting the Home page from markdown to something similar to Event Page

### How are you accomplishing it?
- Schema migration

### Is their anything reviewers should know?
- I haven't added the Image column in the table yet since storing images in the table itself will not be a good idea. 
- I was thinking of using ActiveStorage provided by Rails. We can store files to Microsoft Azure or AWS. 
- I kind of like separating the schema migration and so will have a separate MR for ActiveStorage setup (Only if reviewer have any other Idea, will follow that)

- [x] Is it safe to rollback this change if anything goes wrong?

### CODE REVIEW

**Comment by amanpatel123** [comment]
@EricPickup Any comments on using ActiveStorage?

File: db/schema.rb [hunk 1]
```diff
@@ -65,6 +65,14 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "news_posts", force: :cascade do |t|
+    t.string "title"
+    t.date "occurred_at"
```
Comment (EricPickup) [inline db/schema.rb:70]: what is this for?

**Comment by amanpatel123** [comment]
Well my bad I was under an impression that AWS s3 bucket was free for small usages but guess not

File: db/schema.rb (hunk 1 above)
Comment (amanpatel123) [reply db/schema.rb:70]: Ther is a date attached to every News. 
image

Comment (amanpatel123) [reply db/schema.rb:70]: I completely forgot that that can go in the description as markdown text field

Comment (EricPickup) [reply db/schema.rb]: @amanpatel123 we could just use the created_at date

**Pull request merged**